inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/workflow/group/CreateGroupWorkflowDefinition.java

```diff
@@ -41,61 +41,61 @@
 public class CreateGroupWorkflowDefinition implements WorkflowDefinition {
 
     @Autowired
-    private GroupInitProcessListener groupInitProcessListener;
+    private InitGroupListener initGroupListener;
     @Autowired
-    private GroupCompleteProcessListener groupCompleteProcessListener;
+    private InitGroupCompleteListener initGroupCompleteListener;
     @Autowired
-    private GroupFailedProcessListener groupFailedProcessListener;
+    private InitGroupFailedListener initGroupFailedListener;
     @Autowired
     private GroupTaskListenerFactory groupTaskListenerFactory;
 
     @Override
     public WorkflowProcess defineProcess() {
-
         // Configuration process
         WorkflowProcess process = new WorkflowProcess();
-        process.addListener(groupInitProcessListener);
-        process.addListener(groupCompleteProcessListener);
-        process.addListener(groupFailedProcessListener);
-
-        process.setType("Group Resource Creation");
         process.setName(getProcessName().name());
+        process.setType(getProcessName().getDisplayName());
         process.setDisplayName(getProcessName().getDisplayName());
         process.setFormClass(GroupResourceProcessForm.class);
         process.setVersion(1);
         process.setHidden(1);
 
+        // Set up the listener
+        process.addListener(initGroupListener);
+        process.addListener(initGroupCompleteListener);
+        process.addListener(initGroupFailedListener);
+
         // Start node
         StartEvent startEvent = new StartEvent();
         process.setStartEvent(startEvent);
 
-        // init MQ
+        // Init MQ
         ServiceTask initMQTask = new ServiceTask();
-        initMQTask.setName("initMQ");
+        initMQTask.setName("InitMQ");
```

Comment:
make it enum

Reply:
Good idea.
I'll change them to Enum in the next PR.